packages/dds/merge-tree/src/test/client.searchForMarker.spec.ts

```diff
@@ -127,36 +74,36 @@ describe("TestClient", () => {
 			});
 			assert.equal(client.getLength(), 8);
 
-			const tile1 = client.searchForMarker(4, "Eop", true);
+			const marker1 = client.searchForMarker(4, "Eop", true);
 
-			assert(tile1, "Returned tile undefined.");
+			assert(marker1, "Returned marker undefined.");
 
 			let exp = client.mergeTree.referencePositionToLocalPosition(
-				tile1,
+				marker1,
 				UniversalSequenceNumber,
 				client.getClientId(),
 			);
 
-			assert.equal(exp, 4, "Tile with label not at expected position");
+			assert.equal(exp, 4, "Marker with label not at expected position");
 
-			const tile2 = client.searchForMarker(4, "Eop", false);
+			const marker2 = client.searchForMarker(4, "Eop", false);
 
-			assert(tile2, "Returned tile undefined.");
+			assert(marker2, "Returned marker undefined.");
 
 			exp = client.mergeTree.referencePositionToLocalPosition(
-				tile2,
+				marker2,
 				UniversalSequenceNumber,
 				client.getClientId(),
 			);
 
-			assert.equal(exp, 4, "Tile with label not at expected position");
+			assert.equal(exp, 4, "Marker with label not at expected position");
 		});
 
-		it("Should be able to find non preceding tile based on label", () => {
-			const tileLabel = "EOP";
+		it("Should be able to find non preceding marker based on label", () => {
```

Comment:
Let's drop the "[non ]preceding" terminology now that we no longer use it and just use forwards and backwards.